bot.py

```diff
@@ -1,15 +1,184 @@
+        @self.bot.on.private_message(text=["консультации у <teacher>",
+                                           "консультации"])
+        async def any_message(message: Message, teacher: Optional[str] = None):
+            await message.answer("подождите пожалуйста 5 секунд...")
+            if teacher is not None:
+                consultation = self.api.get_consultations_by_teacher(teacher)
+            else:
+                consultation = self.api.get_consultations()
+            if consultation:
+                try:
+                    await message.answer(consultation)
+                except:
+                    await message.answer(consultation[:900])
+            elif teacher:
+                await message.answer("нет такого учителя, которого вы указали.")
+            else:
+                await message.answer("нет данных о консультациях.")
+
+        # --------------------------------------------------------------------------------------------------------------------
+        #                                               Teacher searching
+        # --------------------------------------------------------------------------------------------------------------------
+        @self.bot.on.chat_message(text=["-thk?список учителей",
+                                        "-thk?учителя",
+                                        "-thk?поиск учителя <teacher>",
+                                        "-thk?поиск <teacher>",
+                                        "-thk?учитель <teacher>"])
+        @self.bot.on.private_message(text=["список учителей",
+                                           "учителя",
+                                           "поиск учителя <teacher>",
+                                           "поиск <teacher>",
+                                           "учитель <teacher>"])
+        async def any_message(message: Message, teacher: Optional[str] = None):
+            await message.answer("подождите пожалуйста 5 секунд...")
+            if teacher is not None:
+                search = self.api.search_teacher_by_name(teacher)
+            else:
+                search = self.api.search_teachers()
+            if search:
+                try:
+                    await message.answer(search)
+                except:
+                    await message.answer(search[:900])
+            elif teacher:
+                await message.answer("нет такого учителя, которого вы указали.")
+            else:
+                await message.answer("нет данных об учителях.")
+
+        # --------------------------------------------------------------------------------------------------------------------
+        #                                               Command list
+        # --------------------------------------------------------------------------------------------------------------------
+        @self.bot.on.chat_message(text=["-thk?список команд",
```

Comment:
Command list is useless, when we use keyboard. Keyboard is added to solve this problem.